### PR TITLE
copy non xsl files from data/xsl to the html dirs.

### LINKED ISSUE #688: Automate copying of non-xsl files into target scoreboard directories
**Is your feature request related to a problem?** 

Yes.  When the scoreboard generates the html there are
needed files .css and .png files.  Those files must
be manually copied.

**Feature Description**: 

For all non .xsl files in the data/xml copy those files into
the html and public_html directories, same directories as
the directories where the html files are created.

**Have you considered other ways to accomplish the same thing?** 

Manually done.

**Do you have any specific suggestions for how your feature would be ***implemented*** in PC^2?**

Each time the html is generated, copy the non xsl files.

Consider not copying the files if the files are identical.
For exmaple
02/04/2023  07:42 PM             2,070 standings.css

if the file in xsl is a differnt time or size, then update the standings.css
02/04/2023  07:55 PM             2,555 standings.css

**Additional context**:

### PR DESCRIPTION
**Pull request by troy2914**
with some intelligence.

eg into ./html and ./public_html by default.

fixes #688

### Precise steps for _testing_ the PR (i.e., how to demonstrate that it works correctly)
start server loading a contest from samps/contests
start board
you should see the standing.css that is in data/xsl be copied to html and public_html directories.

### CODE REVIEW

File: src/edu/csus/ecs/pc2/ui/board/ScoreboardCommon.java [hunk 1]
```diff
@@ -102,6 +189,11 @@ public void generateOutput(String xmlString, String xslDir, String outputDir, Lo
                 } catch (Exception e) {
                     log.log(Log.WARNING, "Trouble transforming " + xslFilename, e);
                 }
+            } else {
+                if (new File(xslDir + File.separator+ xslFilename).isFile()) {
+                    // not xsl, and a file, so just copy it**
```
Comment (johnbrvc) [inline src/edu/csus/ecs/pc2/ui/board/ScoreboardCommon.java:194]: What does the trailing ** mean?  "copy it**" ?

**Review by johnbrvc — APPROVED** [review]
Looks good to me.
Seems to work.  Maybe the `index.xsl `should be changed to include the `standings.css` (link href ...) instead of "including the standings.css" file contents, like it does now.  Since we have this feature, might as well use it.

File: src/edu/csus/ecs/pc2/ui/board/ScoreboardCommon.java (hunk 1 above)
Comment (troy2914) [reply src/edu/csus/ecs/pc2/ui/board/ScoreboardCommon.java:194]: ** means only if needed :)

**Review by johnbrvc — APPROVED** [review]

File: data/xsl/standings.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
+// $Id$
```
Comment (clevengr) [inline data/xsl/standings.css:1]: Isn't the above line a reference to an obsolete source control system?  Does it actually get used somewhere?

**Review by clevengr — APPROVED** [review]
I reviewed the code changes and they all seem to make sense; I started a Server loaded with a sample contest and submitted several runs (some no, some yes) and observed that the `standings.css` file was copied into both the `html` and `public_html` folders, and that the`index.html` files in both those folders used the appropriate .css coloring for No, Pending, and Yes.  I approved the PR.

**Pull request merged**